README.md

```diff
@@ -11,23 +11,32 @@ Interface to FreeType to generate font bitmaps run time
 - Add library: `freetype` (for GCC: `-L/usr/local/lib -lfreetype`)
 
 ## Usage in LVGL
+
+To enable cache, set`LV_USE_FT_CACHE_MANGER  1`in lv_freetype.h.
```

Comment:
This is a nitpick, but if we don't fix typos now, they will be in the API for months or years. Please change to `MANAGER`.